spec/index.bs

```diff
@@ -6754,7 +6783,8 @@ Note: {{GPUCompilationMessage}}.{{GPUCompilationMessage/offset}} and
             <div data-timeline=device>
                 [=Device timeline=] |synchronization steps|:
 
-                1. When either of the following events occur (whichever comes first),
+                1. If |this|.{{GPUObjectBase/[[device]]}}.{{device/[[destroy started]]}} is `true`,
+                    or when either of the following events occur (whichever comes first),
```

Comment:
The phrasing here is fairly awkward. I read the last couple of lines (occur or has already occurred) several times convinced that they were redundant before I realized what it's trying to say. That said I'm not sure how to improve it.

I do think putting the `[[destroy started]]` check into the list below rather than as a separate check up here would help, even if it necessitates some rephrasing of this step. Maybe something like this?

>1. If any of the following are `true` or become `true`:
>
>   - |this|.{{GPUObjectBase/[[device]]}}.{{device/[[destroy started]]}} is `true`
>   - The [=device timeline=] is informed that [=shader module creation=] has succeeded or failed for |this|
>   - |this|.{{GPUObjectBase/[[device]]}} [=is lost=]
>
>   Then issue the subsequent steps on <var data-timeline=content>contentTimeline</var>.

Reply:
I don't want to trigger an event when `[[destroy started]]` is set to true, but I agree, I struggled with the wording a lot. How about this (in the latest commit)?